docs/use.html

```diff
@@ -0,0 +1,722 @@
+<table>
+  <tr>
+      <th>Arguments</th>
+      <th></th>
+  </tr>
+  <tr>
+      <td><code>NAME</code></td>
+      <td>The unique name of the parent. The name is used as a reference for other tags.</td>
+  </tr>
+  <tr>
+      <td><code>ORDER</code></td>
+      <td>The placement of the children in relation to their siblings. The order starts with 0 and defaults to alphabetical.</td>
+  </tr>
+</table>
+<p>Example:</p>
+<pre><code>/**
+  * @stylesheet buttons.css Buttons
+  * @parent styles
+  *
+  * Here's an example of our default button:
+  *
+  * @iframe demos/buttons.html
+**/
+
+</code></pre>
+<p>With this configuration the stylesheet <strong>buttons.css</strong> will show under the <strong>Styles</strong> section:</p>
+<p><img src="static/img/style-guide-tag-parent.png"/></p>
+<h3>@group</h3>
+<p>You can use this tag to group pages or stylesheets in the sidenav of your Living Style Guide.</p>
+<table>
+  <tr>
+      <th>Arguments</th>
+      <th></th>
+  </tr>
+  <tr>
+      <td><code>NAME</code></td>
+      <td>The unique name of the group. The name is used as a reference for other tags.</td>
+  </tr>
+  <tr>
+      <td><code>ORDER</code></td>
+      <td>The placement of the children in relation to their siblings. The order starts with 0 and defaults to alphabetical.</td>
+  </tr>
+  <tr>
+      <td><code>TITLE</code></td>
+      <td>The title that will be shown in the sidebar.</td>
+  </tr>
+</table>
+<p>Example:</p>
+<pre><code>@page styles Styles
+@group styles.colors 0 Colors
+@group styles.typography 1 Typography
+@group styles.forms-elements 2 Form Elements
+
+
+This is my style guide.
+
+</code></pre>
+<p>This will create the groups: <strong>Colors</strong>, <strong>Typography</strong>, and <strong>Form Elements</strong> under the sidebar shown for the <strong>Styles</strong> section:</p>
+<p><img src="static/img/style-guide-tag-group.png"/></p>
+<h2>Creating a Page</h2>
+<p>Now that you have a grasp on what tags are used for, let's dive into how to create your Living Style Guide.</p>
+<p>The first step is to create a <strong>page</strong> which will be used as the index of your styleguide. To create this page you can create a file in the <code>styles</code> directory called <code>styles.md</code>, and inside add the tag <a href="use.html#section=section__page">@page</a>. Your code should look something like this:</p>
+<pre><code>@page styles Styles
+
+This is my style guide.
+
+</code></pre>
+<p>Anything after the line with the tag will be used as text on your page.</p>
+<h3>Using Markdown</h3>
+<p>With DocumentCSS you can use <a href="https://en.wikipedia.org/wiki/Markdown">markdown</a> to structure semantically your documention. Markdown is great becuase is a lightweight markup that won't clutter the comments inside of your style sheets. For example, if you want to call out a piece of text to be displayed as &quot;code&quot; in markdown you can simply wrap the text in &quot;`&quot;. For example:</p>
```

Comment:
Also, the " ` " is a little confusing. Might want to just wrap that in a code tag itself.